### ISSUE TITLE
Compile error on checkerboard_detector

### ISSUE THREAD

**Issue by wkentaro** [jsk-ros-pkg/jsk_recognition#1919]
Related to https://github.com/jsk-ros-pkg/jsk_recognition/pull/1893

@mmurooka Could you please fix this? I think we should declare dependency on `XX_gencfg` before building the executable.

```
Errors     << checkerboard_detector:make /home/wkentaro/ros/indigo/logs/checkerboard_detector/build.make.000.log              
/home/wkentaro/ros/indigo/src/jsk-ros-pkg/jsk_recognition/checkerboard_detector/src/checkerboard_detector.cpp:54:62: fatal error: checkerboard_detector/CheckerboardDetectorConfig.h: No such file or directory
 #include <checkerboard_detector/CheckerboardDetectorConfig.h>
                                                              ^
compilation terminated.
The bug is not reproducible, so it is likely a hardware or OS problem.
make[2]: *** [CMakeFiles/checkerboard_detector.dir/src/checkerboard_detector.cpp.o] Error 1
make[1]: *** [CMakeFiles/checkerboard_detector.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
make: *** [all] Error 2
```

**Comment by mmurooka** [comment]
Sorry for trouble. Fixed in https://github.com/jsk-ros-pkg/jsk_recognition/pull/1920.

**Comment by wkentaro** [comment]
Closed via #1920
